x-pack/filebeat/filebeat.reference.yml

```diff
@@ -403,6 +403,18 @@ filebeat.modules:
     # of the document. The default is true.
     #var.use_namespace: true
 
+#------------------------------ Palo_alto Module ------------------------------
+- module: palo_alto
+  log:
```

Comment:
Yes, makes sense.

However, I chose pan_os instead of pan-os, for a few reasons:
- It makes sense to keep the custom fields in a subgroup, from `palo_alto.*` to `palo_alto.pan-os.*`. In this case, the `pan-os` component in a field name causes a lot of trouble with painless, it complicates `if` conditions because you can't do `ctx?.palo_alto?.pan-os` as it is interpreted as a subtraction.
- It looks kind of ugly to mix underscore and hyphen as in `palo_alto.pan-os`
- Other filesets / submodules use underscore
- ECS suggests to use underscore.

On the other side, `pan-os` is the well-known name. WDYT?

Reply:
SGTM. The benefits of `pan_os` outweigh the benefits of `pan-os`.